src/checkbox/base-checkbox.tsx

```diff
@@ -25,11 +25,7 @@ export interface BaseCheckboxProps extends BaseComponentProps, FormFieldControlP
   disabled?: boolean;
 
   /**
-   * Specifies the ID of the native form element. You can use it to relate
-   * a label element's `for` attribute to this control.
-   *
-   * It defaults to an automatically generated ID that
-   * is provided by its parent form field component.
+   * Specifies the ID of the native form element. It uses an automatically generated ID by default.
```

Comment:
@stirache please review this description change, thank you!

Reply:
"Specifies the ID of the native form element. It uses an automatically generated ID by default." --> "Specifies the ID of the native form element. By default, it uses an automatically generated ID."